### ISSUE TITLE
Is a new release forthcoming?

### ISSUE THREAD

**Issue by daneah** [axnsan12/drf-yasg#780]
We're happy users of drf-yasg, and we're currently undertaking the upgrade to Django 3's next LTS release (3.2). I see there has been some work to incorporate Django 3.2 compatibility, but that there hasn't been a new release since then. Do you anticipate cutting such a release soon? Is there anything we can do to help? Thanks!

**Comment by PaulWay** [comment]
Yeah, I'm worried that once again `drf-yasg` is no longer being maintained.  We need to get multiple maintainers for this package and get started on the pull requests and issues.

**Comment by tfranzel** [comment]
In case you were not aware, you might want to have a look at https://github.com/tfranzel/drf-spectacular

migration guide: https://drf-spectacular.readthedocs.io/en/latest/drf_yasg.html

**Comment by JoelLefkowitz** [comment]
@daneah Thanks for raising this. A release has now been made. Over the next few weeks I'm aiming to process all outstanding PRs. The best way to help is to review an existing PR and add comments / approve the work. I am taking responsibility to keep future releases up to date and more frequent. Thank you!
